### PR TITLE
Refactor grains mentoring.md

### PR DESCRIPTION
**Pull request by AlexMisiulia**
Replace test operator with arithmetic context, exit with return

### CODE REVIEW

File: tracks/bash/exercises/grains/mentoring.md [hunk 1]
```diff
@@ -13,13 +13,11 @@ function main() {
...
   if [[ "$square" == "total" ]]; then
     echo "2^64 - 1" | bc
-    exit 0
-  elif [[ "$square" -lt 1 || "$square" -gt 64 ]]; then
+  elif (( square < 1 || square > 64 )); then
     echo "Error: invalid input"
-    exit 1
+    return 1
   else
     echo "2^($square - 1)" | bc
```
Comment (IsaacG) [inline tracks/bash/exercises/grains/mentoring.md:20]: Want to incorporate herestrings while you are at it?

**Review by IsaacG — APPROVED** [review]

File: tracks/bash/exercises/grains/mentoring.md (hunk 1 above)
Comment (AlexMisiulia) [reply tracks/bash/exercises/grains/mentoring.md]: @IsaacG Sure, done

File: tracks/bash/exercises/grains/mentoring.md [hunk 2]
```diff
@@ -39,15 +37,13 @@ like that:
 function main() {
   square="$1"
 
-  if [[ "$square" == "total" ]]; then
+  if [[ "$square" == 'total' ]]; then
```
Comment (IsaacG) [inline tracks/bash/exercises/grains/mentoring.md:40]: Note: The use of double quotes is perfectly fine here and somewhat more consistent. Either way is fine but I'm not sure there's a good argument in favor of making this change.

**Review by IsaacG — APPROVED** [review]
Looks good to me! Thanks for the update.

File: tracks/bash/exercises/grains/mentoring.md (hunk 2 above)
Comment (AlexMisiulia) [reply tracks/bash/exercises/grains/mentoring.md:40]: As I know it's better to use single quotes where you don't need variables expansions

Comment (IsaacG) [reply tracks/bash/exercises/grains/mentoring.md:40]: I'm unaware of any guidance to that effect :) If you're aware of anything like that in any style guides, I'd love to see it.

When strings contain special/meta chars, then single quotes are quite useful. For regular strings, less so.

Comment (AlexMisiulia) [reply tracks/bash/exercises/grains/mentoring.md:40]: I checked once again code styles and articles and didn't find anything except few examples in [Google code style](https://google.github.io/styleguide/shellguide.html#quoting), but they're inconsistent (few first examples use single quotes, other double quotes). But it's easier for me to read literal string when I saw  single quotes and I know that there are no expansions at all. Seems that this is taste question:) Thanks for pointing out.

**Pull request merged**